privacyidea/lib/smsprovider/FirebaseProvider.py

```diff
@@ -150,12 +152,22 @@ def parameters(cls):
                       FIREBASE_CONFIG.APP_ID: {
                           "required": True,
                           "description": _(
-                              "The App ID, that the client should use. Get it from your Firebase console.")
+                              "The App ID, that the Android client should use. Get it from your Firebase console.")
                       },
                       FIREBASE_CONFIG.API_KEY: {
                           "required": True,
                           "description": _(
-                              "The API Key, that the client should use. Get it from your Firebase console.")
+                              "The API Key, that the Android client should use. Get it from your Firebase console.")
+                      },
+                      FIREBASE_CONFIG.APP_ID_IOS:{
+                          "required": True,
```

Comment:
Should we really require the user to create a firebase app for Android **and** iOS? Would it be possible to use only Android **or** iOS?

Reply:
Logically we could only tell *after* the firebase provider is created, that **either** iOS or Andoird parameters are required.

It does not do any harm, if you simply copy values from android to ios or vice versa.

But there might also be "enough" scenarios, where **both** values are needed.

Reply:
Hi @plettich I added this in 9f7f3fc2